samples/bluetooth/peripheral_hids_keyboard/README.rst

```diff
@@ -159,7 +165,8 @@ To test with `nRF Connect for Desktop`_, complete the following steps:
 
    Observe that **LED 3** turns off.
 #. Disconnect the device in nRF Connect.
-   Observe that no new notifications are received and the device is advertising.
+   Observe that no new notifications are received.
+#. Press **Button 4** to continue advertising.
```

Comment:
Maybe it is worth to add here that the user need to press this button when advertising didn't start automatically. Because this sentence is true for chips which support with NFC support or when CONFIG_NFC_OOB_PAIRING is enabled(it is enabled by default) in the other case the advertising will start automatically after a disconnection